Near/near-ref-finance/src/mappings/mappingHandlers.ts

```diff
@@ -0,0 +1,44 @@
+import { Swap, Token, Pool } from "../types";
+import { NearAction } from "@subql/types-near";
+
+export async function handleAction(action: NearAction): Promise<void> {
+  // An Action can belong to either a transaction or a receipt
```

Comment:
I think it's good practice to keep a logger event here to clearly show to people that "this is working"
```suggestion
  logger.info(`Swap found at block ${action.transaction.block_height}`);
  // An Action can belong to either a transaction or a receipt
```